pbft/pbft.cpp

```diff
@@ -474,8 +522,22 @@ pbft::do_committed(const std::shared_ptr<pbft_operation>& op)
         this->broadcast(this->wrap_message(msg));
     }
 
-    // Get a new shared pointer to the operation so that we can give pbft_service ownership on it
-    this->io_context->post(std::bind(&pbft_service_base::apply_operation, this->service, this->find_operation(op)));
+    // TODO: this needs to be refactored to be service-agnostic
+    if (op->request.type() == PBFT_REQ_DATABASE)
+    {
+        // Get a new shared pointer to the operation so that we can give pbft_service ownership on it
```

Comment:
This comment was meant to be removed in one of my commits a while ago; I think it got accidentally readded by a rebase and it doesn't make sense anymore.